app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -357,9 +357,6 @@ public void onDestroy() {
         if (Pokefly.isRunning()) {
             stopService(new Intent(MainActivity.this, Pokefly.class));
         }
-        if (screen != null) {
-            screen.exit();
-        }
```

Comment:
Hmmm... I thought I had asked @thearaks to remove this when he added Pokefly.isRunning(). I guess not.